doc/controls/CardAndCardContentControl.md

```diff
@@ -0,0 +1,120 @@
+# Card & CardContentControl
+
+## Summary
+
+Card represents a control identifiable as a single, contained unit used to visually group related child content and actions that relate information about a subject.
+A card’s layout and dimensions depend on its contents.
+
+## Remarks
+
+Currently, there are three [Material](https://m3.material.io/components/cards/) styles for `Card` and `CardContentControl` that you can use. 
+Depending on the amount of user attention you want to draw to the content you can use:
+- `ElevatedCardStyle` or `ElevatedCardContentControlStyle` to add a subtle z-axis elevation.
+- `FilledCardStyle` or `FilledCardContentControlStyle` to display a simple background color without any elevation or border for the card.
+- `OutlinedCardStyle` or `OutlinedCardContentControlStyle` to display a simple solid stroke along the border of the card.
+
+## Card
+The Card control comes with all the build-in properties deriving from `Control` but also has these additional properties listed below that you can use depending on the content that you want to display:
+
+| Property                  | Type              |
+|---------------------------|-------------------|
+| HeaderContent             | Object            |
+| HeaderContentTemplate     | DataTemplate      |
+| SubHeaderContent          | Object            |
+| SubHeaderContentTemplate  | DataTemplate      |
+| AvatarContent             | Object            |
+| AvatarContentTemplate     | DataTemplate      |
+| MediaContent              | Object            |
+| MediaContentTemplate      | DataTemplate      |
+| SupportingContent         | Object            |
+| SupportingContentTemplate | DataTemplate      |
+| IconsContent              | Object            |
+| IconsContentTemplate      | DataTemplate      |
+| Elevation                 | Double            |
+| ShadowColor               | Windows.UI.Color  |
+
+If you want to have something more custom for the entire content you can use  [CardContentControl](#cardcontentcontrol) instead of using `Card`.
+
+### Usage for Card
+
+#### ElevatedCardStyle
```

Comment:
you can merge the 3 example into a single one
and replace the #### headers with \<-- xml comments -->

Reply:
same for cardcontentcontrol below